app/views/results/common/_annotations.js.erb

```diff
@@ -410,6 +413,13 @@
     });
     updatePreview('new_annotation_content', 'annotation_preview');
     modal.open();
+
+    $('#submit_annotation').click(() => {
```

Comment:
Remove this---by making the checkbox more explicit (see above comment), we can drop the `confirm` altogether (and remove the existing string, or adapt it to the new checkbox label).

Also I think adding this caused the submit button to no longer close the modal? You should look into that more if removing this code still doesn't fix that issue.